pkg/worker/k8spodtemplating.go

```diff
@@ -91,38 +91,43 @@ func sanitizePodName(name string) string {
 }
 
 func getOwnerReferences() []metav1.OwnerReference {
-	var (
-		enabled   bool
-		name, uid string
-	)
-	if err := env.BindMultiple(map[any]string{
-		&enabled: "WHARF_KUBERNETES_OWNER_ENABLE",
-		&name:    "WHARF_KUBERNETES_OWNER_NAME",
-		&uid:     "WHARF_KUBERNETES_OWNER_UID",
+	var enabled bool
+	if err := env.Bind(&enabled, "WHARF_KUBERNETES_OWNER_ENABLE"); err != nil {
+		log.Warn().WithError(err).Message("Failed binding WHARF_KUBERNETES_OWNER_ENABLE environment variables.")
```

Comment:
```suggestion
		log.Warn().WithError(err).Message("Failed binding WHARF_KUBERNETES_OWNER_ENABLE environment variable.")
```

Reply:
Oops sorry I missed this one. I'm applying the changes in #182 instead